src/tests.rs

```diff
@@ -54,7 +60,7 @@ impl Interface for TestInterface {
         Ok(())
     }
 
-    fn get_data(&self, _: &str) -> Result<Bytecode> {
+    fn raw_get_data(&self, _: &str) -> Result<Vec<u8>> {
```

Comment:
if we want to be consistent, both the `key` parameter and the return value should be `Vec<u8>`

Reply:
What do you think about it @adrien-zinger having keys (memory addresses) encoded as Integer (or bytes) rather than Strings?

Reply:
I don't know... encoding the key in the database is done in massa node for the moment... in the runtime the ABI is like `set_data("cryptic_key_or_not", "data")` It doesn't really change the main functionality... as we want.

The slice used to be a string in the smart contract, we are the prisoners of that famous _StringPtr_ :rofl: . We can just use a `.to_be_bytes` here and expose a `vec<u8>` in the interface if we want to be _clean clean clean_, and we're done?

Reply:
I created another issue to discuss that since again it's not related with this PR!